src/convertToObject.js

```diff
@@ -12,7 +12,12 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const s = sourceString
+    .split(';')
+    .map(e => e.split(':').map(x => x.trim()))
```

Comment:
```suggestion
    .map(e => e.split(':')
    .map(x => x.trim()))
```
be consistent

Reply:
it's inside first method, does this make it better?